scripts/gis-material-update/README.md

```diff
@@ -323,6 +324,43 @@ Output files (names configured in `config.yaml`)
 
 - tormays_central_business_areas.gpkg
 
+### `liikennevaylat`
+
+Prerequisite: 
+- `central_business_area` material fetched
+- `ylre_katuosat` material fetched
+- `liikennevaylat` material fetched
+- `central_business_area` material processed
+- `ylre_katuosat` material processed
+
+In gis-material-update/.env file there need to be following secrets:
```

Comment:
Mention about new envs (`HELSINKI_EXTRANET_USERNAME` and `HELSINKI_EXTRANET_PASSWORD`) should be earlier in README.md because they are already needed when running `docker-compose run --rm gis-fetch liikennevaylat`